GitExtUtils/GitUI/Theming/AppColor.cs

```diff
@@ -11,7 +11,7 @@ public enum AppColor
     {
         OtherTag,
         AuthoredHighlight,
-        HighlightAllOccurences,
```

Comment:
This will affect all themes too (the .css files)

If changed, it should be applied to 3.5 too, there are maybe not so many external themes yet

Reply:
I wonder if we can use the public API analyzer to catch these kinds of public API changes. It would have made me more reluctant to make this change if I saw that it was modifying a documented public API.

Reply:
There are tests setup that fail due to this, so there is some catch...

Reply:
Yes, there are PublicApi analyzers, that we use in dotnet/winforms.

It is on my (very long) TODO list, but it'll have a great impact once we'll have split the API contracts out of this repo.

Reply:
I reverted this change and rebased on master.